examples/postgres-secret.yaml

```diff
@@ -9,4 +9,4 @@ stringData:
   POSTGRES_PORT: "5432"
   POSTGRES_USER: postgres
   POSTGRESQL_ADMIN_PASSWORD: admin123
-  POSTGRES_HOST: backstage-psql
\ No newline at end of file
+  POSTGRES_HOST: backstage-psql-bs1
```

Comment:
In the previous PR (https://github.com/janus-idp/operator/pull/53/files#r1422807856), you were dynamically injecting a `POSTGRES_HOST` env var with the actual DB service name.
I guess you plan to work on that as part of https://github.com/janus-idp/operator/issues/63, right?

Reply:
Yes this will be done in #63 instead.